epochStart/bootstrap/storageProcess.go

```diff
@@ -33,10 +33,11 @@ import (
 // from storage
 type ArgsStorageEpochStartBootstrap struct {
 	ArgsEpochStartBootstrap
-	ImportDbConfig             config.ImportDbConfig
-	ChanGracefullyClose        chan endProcess.ArgEndProcess
-	TimeToWaitForRequestedData time.Duration
-	ChainRunType               common.ChainRunType
+	ImportDbConfig                       config.ImportDbConfig
+	ChanGracefullyClose                  chan endProcess.ArgEndProcess
+	TimeToWaitForRequestedData           time.Duration
+	ChainRunType                         common.ChainRunType
```

Comment:
why do we still have to provide the ChainRunType?

Reply:
Will remove chainRunType in one of the following PR-s. This PR is for extracting functionality in Creators/Factories without changing the current logic.

Reply:
great